### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Redsandro**
Added installation and callback info

### CODE REVIEW

**Comment by Redsandro** [comment]
Let me add myself before you merge this in. :smile_cat:

File: README.md [hunk 1]
```diff
@@ -4,15 +4,40 @@ Node.JS module for converting XML documents into JSON objects. It should be one
 of the fastest converters available. Uses [RapidXML](http://rapidxml.sourceforge.net/).
 Inspired by [fast-feed](https://github.com/rla/fast-feed).
 
+## Installation
+
+`npm install --save rapidx2j`
+
+If your `gyp` is out of date, you might run into the following error:
```
Comment (damirn) [inline README.md]: is this really important or specific to rapidx2j?

**Comment by Redsandro** [comment]
@damirn 

> is this really important or specific to rapidx2j?

It's theoretically not specific, but I have never ever had this except with `rapidx2j` and it took me an hour to figure out the silly thing. I thought I'd save some people an hour.

Should I remove this?

**Comment by damirn** [comment]
Remove it from now, we have never seen this issue, but if more people report it, i'll let you added it back.

**Comment by Redsandro** [comment]
@damirn removed.

**Pull request merged**